layercatalog.json

```diff
@@ -1272,5 +1272,56 @@
         "native_crs": "EPSG:4326"
       }
     }
+  },
+  {
+    "id": "COP_DEM_GLO_30M_COG",
```

Comment:
We'll need a better name for openEO.
Can we use 'COPERNICUS_30'? That aligns it with the Sentinelhub name. Details like 'COG' are not relevant anymore when using openEO.

Reply:
Done.